securedrop_client/gui/widgets.py

```diff
@@ -3238,64 +3752,7 @@ def on_reply_sent(self, source_uuid: str, reply_uuid: str, reply_text: str) -> N
         self.reply_flag = True
         if source_uuid == self.source.uuid:
             self.add_reply_from_reply_box(reply_uuid, reply_text)
-
-
-class SourceConversationWrapper(QWidget):
```

Comment:
I wish the diff was easier to review instead of it looking like `SourceConversationWrapper` was deleted and added as new; would be nice to see just the changes

Reply:
I guess a lot has changed

Reply:
I can move it back.

Reply:
if you do, great, but if you don't, that's fine because i reviewed the changes as is and didn't spot any issues with the refactor.